### PR TITLE
Bug/HomeSignIn-page-issues

### PR DESCRIPTION
**Pull request by PaulMEdwards**
Please review/merge #56 first.

- [x] Login Email & Password input fields should both have `name` attributes
- [x] Login Form should support Enter to submit
- [x] Buttons on page need icons

### CODE REVIEW

**Comment by PaulMEdwards** [comment]
[Bug: HomeSignIn page issues](https://trello.com/c/hgvyxAOe/37-bug-homesignin-page-issues)

**Review by Brodt7258 — APPROVED** [review]
good call adding the form wrapper. accessibility and password managers are good

File: .gitignore [hunk 1]
```diff
@@ -26,6 +26,9 @@ api/emails/templates/placeholder.html
 .env.test.local
 .env.production.local
 
+package-lock.json
+yarn.lock
```
[the remark this answers is not in the document]
Comment (PaulMEdwards) [reply .gitignore]: Bad info on my part. Re-added.

**Review by william-jensen01 — COMMENTED** [review]
Looks good. Thanks for adding the name tags to the input fields for email and password!

**Review by william-jensen01 — APPROVED** [review]

**Pull request merged**